src/cmd/flux-jobs.py

```diff
@@ -320,6 +320,16 @@ def parse_args():
         metavar="KEY,...",
         help="Specify sort order",
     )
+    parser.add_argument(
+        "-w",
+        "--width",
+        type=int,
+        default=-1,
+        const=0,
+        metavar="N",
+        nargs="?",
+        help="Truncate output at width N. If N=0 (default), use terminal size",
```

Comment:
This makes it sound like `N=0` is the default, vs the "no arg default".  I didn't know how we normally word it,  `flux-jobs` only other optional arg option is:

`        help="Colorize output; WHEN can be 'never', 'always', or 'auto' (default)",`

maybe just remove the "(default)"?  Perhaps say "(default no truncation)" at the end?

Reply:
How about 

> Truncate output at width N. If N=0 (default if N omitted), use terminal size.